crates/sui-open-rpc/src/examples.rs

```diff
@@ -1406,35 +1399,44 @@ impl RpcExampleProvider {
     fn suix_get_stakes(&mut self) -> Examples {
         let principal = 200000000000;
         let owner = SuiAddress::from(ObjectID::new(self.rng.gen()));
-        let result = DelegatedStake {
-            validator_address: SuiAddress::from(ObjectID::new(self.rng.gen())),
-            staking_pool: ObjectID::new(self.rng.gen()),
-            stakes: vec![
-                Stake {
-                    staked_sui_id: ObjectID::new(self.rng.gen()),
-                    stake_request_epoch: 62,
-                    stake_active_epoch: 63,
-                    principal,
-                    status: StakeStatus::Active {
-                        estimated_reward: (principal as f64 * 0.0026) as u64,
+        let result = vec![
+            DelegatedStake {
+                validator_address: SuiAddress::from(ObjectID::new(self.rng.gen())),
+                staking_pool: ObjectID::new(self.rng.gen()),
+                stakes: vec![
+                    Stake {
+                        staked_sui_id: ObjectID::new(self.rng.gen()),
+                        stake_request_epoch: 62,
+                        stake_active_epoch: 63,
+                        principal,
+                        status: StakeStatus::Active {
+                            estimated_reward: (principal as f64 * 0.0026) as u64,
```

Comment:
Confused about this estimated_reward, is it just a random value?

Reply:
Yes, I think so -- these are just made up examples.